kedro/templates/project/hooks/utils.py

```diff
@@ -15,6 +15,9 @@
 # Configuration key for documentation dependencies
 docs_pyproject_requirements = ["project.optional-dependencies"]  # For pyproject.toml
 
+# Requirements for example pipelines
+example_pipeline_requirements = "seaborn~=0.12.1\nscikit-learn~=1.0\n"
```

Comment:
This approach seems somewhat risky, as we are hardcoding the complete version names for all the libraries slated for deletion. I'm concerned about how we will remember to update these if we decide to modify the version requirements in the future. This is also for lines 8-12.

Reply:
Valid point I had the same feelings but we'll need to look for a more robust way to do it, maybe with regex?

Reply:
I can create a follow up issue on this.

Reply:
I agree, we need to think more about it, issue is a good point 👍

Reply:
Done.
https://github.com/kedro-org/kedro/issues/3432